src/provisioner/manifest.py

```diff
@@ -0,0 +1,67 @@
+# CORTX Python common library.
```

Comment:
Rename this file to release.py because that is the primary purpose of it.

Reply:
done.